modules/accessing-inference-endpoints-for-models-deployed-on-single-model-server.adoc

```diff
@@ -95,5 +95,9 @@ curl -ks <inference_endpoint_url>/v2/models/<model_name>/infer -d '{ "model_name
 --
 endif::[]
 
-//[role='_additional-resources']
-//.Additional resources
+[role='_additional-resources']
+== Additional resources
```

Comment:
For each of these links, let's include a caret (^) symbol at the end of the link title. This instructs the web browser to open the link in a new tab.